tests/cases/dom/events/mediaQuery.js

```diff
@@ -0,0 +1,195 @@
+import {mediaQuery} from "../../../../dom/events";
+import QUnit from "qunit";
+import {findOne} from "../../../../dom/traverse";
+import {createElement} from "../../../../dom/manipulate";
+import {setStyles} from "../../../../dom/css";
+
+const matchQueryList = [];
+
+QUnit.module("dom/events/mediaQuery()", {
+    beforeEach: () => {
+        findOne("#qunit-fixture").innerHTML = `
+            <div class="example"></div>
+        `;
+    },
+    afterEach: () => {
+        matchQueryList.forEach((element, index) => {
+            if (undefined !== element.mediaQuery)
+            {
+                if (undefined !== element.mediaQuery.remove)
+                {
+                    element.remove();
+                }
+
+                if (undefined !== element.mediaQuery.targetWindow)
+                {
+                    element.targetWindow.close();
+                }
+            }
+
+            matchQueryList.splice(index, 1);
+        });
+    }
+});
+
+
+QUnit.test(
+    "mediaQuery() on window media query matches without immediate execution",
```

Comment:
```suggestion
    "mediaQuery() on window media query doesn't match without immediate execution",
```
?

Reply:
No, this should match, but should not call the callback immediately.